assets/js/modules/analytics/datastore/constants.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Analytics Constants.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const STORE_NAME = 'modules/analytics';
+
+export const ACCOUNT_CREATE = 'account_create';
```

Comment:
Documenting these, even briefly, would be handy. I know what they do, but I had to recall it and I'm sure we'll eventually forget 😉

Reply:
✅